Documentation/user-manual.txt

```diff
@@ -642,7 +642,7 @@ you might like to call one "Alice's Suunto D4" and the other
 dive computers of the same model, the logs of both (or all) being uploaded.
 In this case it might be prudent to call one of them
 "Suunto D4 (1)" and another "Suunto D4 (2)". This is easily done in _Subsurface_.
-On the *Main Menu*, select _Log ->  Edit device names_. A dialog box opens, showing the
+On the *Main Menu*, select _Log ->  Edit device names_. A dialogue box opens, showing the
```

Comment:
I think we agreed (I mean, I simply asked everyone to agree) that we'll stick with simplified (also known as US American) spelling...

Reply:
Done